static/js/topic_page/map_tile.tsx

```diff
@@ -121,141 +112,143 @@ export function MapTile(props: MapTilePropType): JSX.Element {
       )}
     </div>
   );
-}
```

Comment:
It's better to put helper functions outside of the react component when possible so that they're not re-created on every re-render. Since these helper functions can exist outside of the component, I think we should keep them outside of the component.

Reply:
That's a good point!